### PR TITLE
Added support for yolact auto-build

### PR DESCRIPTION
**Pull request by rahul-tuli**
The goal of this PR is to auto-build a `yolact` wheel

* Add a `build_wheel.sh` bash script to build a yolact wheel and test installation
* Add a `setup.py` file to guide building process
* Add a `MANIFEST.in` for assets to be included in wheel
* Add a GHA Workflow for auto-build nightly
* Add a GHA Workflow for auto-building release branch

TODO: 
- [x] ~Fix Relative imports bug caught by @KSGulin~
- [x] ~Remove `auto-build` branch from GHA, only keep master~
- [x] ~Setup release tags/Setup an automated workflow for this~

### CODE REVIEW

**Review by KSGulin — CHANGES_REQUESTED** [review]
Great job on this Rahul! I like the clean implementation.

The only part I can see missing right now is the relative import updates. I.e. replacing imports in between directories, such as the `from data import ...` found in utils. I primarily saw `data` and `layers` imports that need to be updated, but I didn't do an exhaustive search, so there may be more.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+def read_requirements():
+    build_dir = os.path.abspath(os.path.dirname(__file__))
+    with open(os.path.join(build_dir, "yolact", "requirements.txt")) as f:
+        return f.read().splitlines()
+
+
+def packages():
+    return [
+        'yolact',
```
Comment (bfineran) [inline setup.py:13]: not an issue currently, but noting that nested directories/modules will not be picked up

**Comment by KSGulin** [comment]
@rahul-tuli actually found some more relative import issues. Files that attempt to import from the root directory need to be updated as well. e.g. https://github.com/neuralmagic/yolact/blob/68ea8f6edcc0d61047a95071fa22d8d271164605/data/config.py#L1

**Comment by rahul-tuli** [comment]
> @rahul-tuli actually found some more relative import issues. Files that attempt to import from the root directory need to be updated as well. e.g.
> 
> https://github.com/neuralmagic/yolact/blob/68ea8f6edcc0d61047a95071fa22d8d271164605/data/config.py#L1

Thanks for catching this Konstantin, the new commits fix this issue! and also print out example import changes for a file and directory

File: setup.py (hunk 1 above)
Comment (rahul-tuli) [reply setup.py]: Addressed!

**Review by KSGulin — APPROVED** [review]

**Pull request merged**